packages/fxa-content-server/app/scripts/templates/sign_in_token_code.mustache

```diff
@@ -1,16 +1,16 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="fxa-signin-code-header">{{#t}}Authorize this sign-in{{/t}}</h1>
+    <h1 id="fxa-signin-code-header">{{#t}}Enter confirmation code{{/t}}<span class="description">{{#t}}for your Firefox account{{/t}}</span></h1>
```

Comment:
Another question regarding how to tag. Should this be a single translation tag? I wonder if breaking this into two parts will hurt the translation's context. There are lots of existing examples in our code where we will use an `{{#unsafeTranslation}}` and include the HTML elements. I assume the reason to do this is so that we don't break up the sentence into multiple translation strings.